typescript/ui/matchHandler.ts

```diff
@@ -33,37 +33,52 @@ export class MatchHandler {
         let positionOrder = ["playerLeft", "playerBottom", "playerRight", "playerTop"]
         _.each(_.zip(this.seatingArrangement.playerOrder, positionOrder), ([playerName, position]) => {
             this.tablePositions[playerName] = position
+        })
+
+        this._currentBidder = undefined
+        this._numberOfBidsMadeThisRound = undefined
+    }
+
+    displayPlayers() {
+        _.each(this.tablePositions, (position, playerName) => {
             let text = playerName;
             if (playerName == this.currentDealer) {
                 text += " (Dealer)"
             };
-            document.getElementById(position + "Label").appendChild(document.createTextNode(text))
+            document.getElementById(position + "Label").innerHTML = text
         })
-
-        this._currentBidder = undefined
-        this._numberOfBidsMadeThisRound = undefined
     }
 
-    start() {
+    startRound() {
         this.match.deal()
+        _.each(this.match.players, player => {
+            player.hand = sortHand(player.hand)
+        })
+        this.displayScores()
+        this.displayPlayers()
         this.displayCards()
         this.collectBidsManually()
     }
 
-    clearCards(): void {
+    clearCards() {
         _.each(this.tablePositions, (position, playerName) => {
             removeChildrenFromElement(document.getElementById(position + "Cards"))
+            removeChildrenFromElement(document.getElementById(position + "PlayedCard"))
         })
     }
 
+    displayScores() {
+        document.getElementById("scores").innerHTML = `Team A: ${this.match.teams["Team A"].score} - Team B: ${this.match.teams["Team B"].score}`
+    }
+
     displayCards() {
         this.clearCards()
 
         _.each(this.seatingArrangement.playerOrder, playerName => {
             let playerObject = this.match.players[playerName]
     
             let cardsDiv = document.getElementById(this.tablePositions[playerName] + "Cards")
-            _.each(sortHand(playerObject.hand), card => {
```

Comment:
If the hand is sorted each time it is displayed, then the suits will be messed up. Once a card is removed, the sorting will still apply and thus only need be done once. If removing a card puts two suits of the same color together, then perhaps there can be a button in the future to resort the hand. Or it can check for that.